lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -314,6 +315,13 @@ LLDBMemoryReader::resolvePointer(swift::remote::RemoteAddress address,
 
 bool LLDBMemoryReader::readBytes(swift::remote::RemoteAddress address,
                                  uint8_t *dest, uint64_t size) {
+  auto [success, _] = readBytesImpl(address, dest, size);
+  return success;
+}
+
+std::pair<bool, bool>
```

Comment:
Interpreting the return value here is hard just by looking at the type. Maybe a typedef or small struct would be useful?